Scientific_workflows/WIT_Metrics/wit_metrics.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
<p>Could you put the functions into a .py file in the same directory and import it? If they are not going to change, no benefit having them in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>